### PR TITLE
remove unneeded extra constructions now that pool allocation is fixed

### PR DESCRIPTION
**Pull request by mbechard**

### CODE REVIEW

File: glslang/MachineIndependent/SymbolTable.cpp [hunk 1]
```diff
@@ -427,11 +427,7 @@ TSymbolTableLevel* TSymbolTableLevel::clone() const
     symTableLevel->retargetedSymbols.clear();
     for (auto &s : retargetedSymbols) {
         // Extra constructions to make sure they use the correct allocator pool
```
Comment (greg-lunarg) [inline glslang/MachineIndependent/SymbolTable.cpp:429]: Can you please fix this comment?

File: glslang/MachineIndependent/SymbolTable.cpp [hunk 2]
```diff
@@ -464,9 +460,7 @@ TSymbolTableLevel* TSymbolTableLevel::clone() const
             continue;
 
         // Need to declare and assign so newS is using the correct pool allocator
```
Comment (greg-lunarg) [inline glslang/MachineIndependent/SymbolTable.cpp:462]: Can you please fix this comment too?

**Review by greg-lunarg — CHANGES_REQUESTED** [review]
Just need to touch up the comments to no longer refer to pool allocation issue. Otherwise good. Thanks!

File: glslang/MachineIndependent/SymbolTable.cpp (hunk 1 above)
Comment (mbechard) [reply glslang/MachineIndependent/SymbolTable.cpp]: Done, sorry about that.

**Comment by greg-lunarg** [comment]
@mbechard If you rebase your branch and re-push it, I think these failures will clear up.

**Comment by mbechard** [comment]
Sorry, forgot recheck the PR to see if the build failed.

**Pull request merged**